system-preparation/6lu7_complex/output/system_4fs.xml

```diff
@@ -295471,6 +295471,5 @@
 			</Exceptions>
 		</Force>
 		<Force forceGroup="0" frequency="25" pressure="1.01325" randomSeed="110257930" temperature="310" type="MonteCarloBarostat" version="1"/>
```

Comment:
Do we want to change the barostat frequency to 50 for more speed, or are we fine with 25?

Reply:
what's 'more speed'? you have numbers on that?

Reply:
I'll change in the future, obviously not now we have stuff running on F@h